plugins/module_utils/foreman_helper.py

```diff
@@ -1541,6 +1539,19 @@ def _recursive_dict_without_none(a_dict, exclude=None):
     return result
 
 
+def _is_resolved(spec, what):
+    if spec.get('type') not in ('entity', 'entity_list'):
+        return True
+
+    if spec.get('type') == 'entity' and (what is None or isinstance(what, dict)):
+        return True
+
+    if spec.get('type') == 'entity_list' and isinstance(what, list) and what and isinstance(what[0], dict):
```

Comment:
It can also be a list of `None`, right?

Reply:
Having a list lookup with `failsafe=True` is probably odd, but i think it can be mixed.

Reply:
We do have code for it ;)

https://github.com/theforeman/foreman-ansible-modules/blob/023c9624efa5cbb69fae1571e35db69a152e3985/plugins/module_utils/foreman_helper.py#L790